Magento2/Sniffs/CodeAnalysis/EmptyBlockSniff.php

```diff
@@ -25,4 +26,21 @@ public function register()
             ]
         );
     }
+    /**
+     * @inheritDoc
+     */
+    public function process(File $phpcsFile, $stackPtr)
+    {
+        $tokens = $phpcsFile->getTokens();
```

Comment:
```
        $tokens = $phpcsFile->getTokens();
        if ($tokens[$stackPtr]['code'] === T_FUNCTION &&
            strpos($phpcsFile->getDeclarationName($stackPtr), 'around') === 0) {
            return;
        }
```

Reply:
If I add the following code to test then the sniff doesn't give warning as per your above comment

`function around() { }`

Reply:
I believe that's fine. It's a really low possibility of such false negative finding.